examples/end-to-end-session-based/01-ETL-with-NVTabular.ipynb

```diff
@@ -69,7 +69,18 @@
    "execution_count": 2,
```

Comment:
<p>I think we call fit transform twice - first above - .fit and .transform and then .compute</p><p> </p><p>and here, we call workflow.fir_transform().to_parquet.</p><p> </p><p>We do the calculation twice.</p><p> </p><p>I think we should change the process above to all fit_transform().to_parquet and then read the data with cudf.read_parquet() here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>